main/createUiDefinition.json

```diff
@@ -458,6 +566,24 @@
           "visible": "[equals(basics('niosModel'), 'vnios-cp-v2200')]",
           "osPlatform": "Linux"
         },
+        {
+          "name": "vmSize-nios-cp-v2205",
+          "type": "Microsoft.Compute.SizeSelector",
+          "label": "Virtual machine size for CP-V2205",
+          "toolTip": "Select the size of VM for CP-V2205",
+          "recommendedSizes": [
+            "Standard_DS3",
+            "Standard_DS3_v2"
```

Comment:
Sizes for both 1405 & 2205 are same. Is this correct?  2205 should have higher capatity than 1405.

Reply:
I have followed the same flavour for CP-1405/2205 as it was done in CP-1400/2200, because it should behave same.

Reply:
From Rahul:
Used same as 1400 & 2200
We are giving both options for 1400 and 2200, customer can choose any one of the two